srml/babe/src/lib.rs

```diff
@@ -151,11 +155,15 @@ decl_module! {
 				.iter()
 				.filter_map(|s| s.as_pre_runtime())
 				.filter_map(|(id, mut data)| if id == BABE_ENGINE_ID {
-					<[u8; VRF_OUTPUT_LENGTH]>::decode(&mut data)
+					<(u64, [u8; VRF_OUTPUT_LENGTH])>::decode(&mut data)
```

Comment:
Can we not put this somewhere in a type? That is used in the node and the runtime? This feels like something that can break very fast in a refactoring.

Reply:
I will switch to `BabePreDigest` here, since I think that the performance penalty is worth it.

Reply:
Which performance penality? Does `BabePreDigest` contains more data?

Reply:
@bkchr Yes, it does.